content/03-originate/originate-sdk.mdx

```diff
@@ -195,56 +209,66 @@ We have another queue, so, as we did before:
 <MultiCodeBlock>
 
 ```typescript
-const token: SecurityToken = await tokenQueue.run();
+const asset: Asset = await assetQueue.run();
 ```
 
 ```javascript
-const token = await tokenQueue.run();
+const asset = await assetQueue.run();
 ```
 
 </MultiCodeBlock>
 
-Implicit in the creation of this token is that Alice, as a private individual, is **both the token's owner and its primary issuance agent (PIA)**. We were satisfied with this situation only up to this point. Now this needs to change.
+Implicit in the creation of this asset is that Alice, as a private individual, is **both the asset's owner and its primary issuance agent (PIA)**. We were satisfied with this situation only up to this point. Now this needs to change.
 
-This `token` instance will not always be in memory, so if we wanted to fetch it at a later date, we would do:
+This `asset` instance will not always be in memory, so if we wanted to fetch it at a later date, we would do:
 
 <MultiCodeBlock>
 
 ```typescript
-const token: SecurityToken = await apiAlice.getSecurityToken({
+const asset: Asset = await apiAlice.assets.getAsset({
     "ticker": "ACME"
 });
 ```
 
 ```javascript
-const token = await apiAlice.getSecurityToken({
+const asset = await apiAlice.assets.getAsset({
     "ticker": "ACME"
 });
 ```
 
 </MultiCodeBlock>
 
-## Secondary keys
+## Secondary Accounts
 
 Now we assume that ACME has gone through CDD and has an account, complete with a private key and its mnemonic, which, in a mirror fashion of that of Alice gives us:
 
 <MultiCodeBlock>
 
 ```typescript
+const signingManagerAcme: LocalSigningManager = await LocalSigningManager.create({
+    "accounts": [{
+        "mnemonic": "word21 word22 ..."
```

Comment:
Maybe: `bottom drive obey lake curtain smoke basket hold race lonely fit walk`

(this is the seed phrase for Alice and friends they add a derivation path onto it "//Alice" -> "bottom drive obey lake curtain smoke basket hold race lonely fit walk//Alice")

(OK to leave as is)

Reply:
I think idea here is to expect the user to use this with mainnet and they would get confused if we specify Alice or its mnemonics. It's better to keep it this way so that they understand. Again leaving as is